substrate/frame/revive/src/wasm/runtime.rs

```diff
@@ -2152,4 +2152,44 @@ pub mod env {
 			already_charged,
 		)?)
 	}
+
+	/// Sets the storage at a fixed 256-bit key with a fixed 256-bit value.
+	/// See [`pallet_revive_uapi::HostFn::set_storage_or_clear`].
+	#[stable]
+	#[mutating]
+	fn set_storage_or_clear(
+		&mut self,
+		memory: &mut M,
+		flags: u32,
```

Comment:
Please sort those two functions next to the other stable storage functions. We want all the stable functions in the front.